src/slices/AccountSlice.ts

```diff
@@ -196,7 +194,7 @@ const accountSlice = createSlice({
       .addCase(calculateUserBondDetails.fulfilled, (state, action) => {
         if (!action.payload) return;
         const bond = action.payload.bond;
-        if (action.payload.interestDue > 0) state.bonds[bond] = action.payload;
```

Comment:
Does this break user claim bond table by showing everything instead of only bonds they have purchased?